docs/validatingwebhookconfiguration.md

```diff
@@ -0,0 +1,7 @@
+# ValidatingWebhookConfiguration Metrics
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| kube_validatingwebhookconfiguration_info | Gauge | `validatingwebhookconfiguration`=&lt;validatingwebhookconfiguration-name&gt; <br> `namespace`=&lt;validatingwebhookconfiguration-namespace&gt; | STABLE |
```

Comment:
New metrics have to be `EXPERIMENTAL`. Apologies for not being clear. We are yet to add documentation to outline this.
```suggestion
| kube_validatingwebhookconfiguration_info | Gauge | `validatingwebhookconfiguration`=&lt;validatingwebhookconfiguration-name&gt; <br> `namespace`=&lt;validatingwebhookconfiguration-namespace&gt; | EXPERIMENTAL |
```